### ISSUE TITLE
User App issues 1->6

### ISSUE THREAD

**Issue by chrisak86** [chrisak86/Salamah-App#3]
1 - The app is saving the location upon 1st usage after installation and always use this location to raise request.**(now location will update Fixed)**
2 - If you change location it shows on the map that location is changed but when a request is raised the source of the request is always the initial one saved.**(Request will be raised from updated location)**
3 - The app is displaying the distance as 103.7 m for everyone using it and ETA 0.45 min for everyone using it. I am testing from Dubai and Madona is testing from Kuwait both for every request we raise we receive the above same ETA and Distance
4 - The compass button on the map is not working**(fixed)**
5- When I raise a ambulance request, only a police request is being raised instead of ambulance + police 
6- When I raise a fire request, only a police request is being raised instead of police + Ambulance + fire

**Comment by tariqarbi03** [comment]
5- When I raise a ambulance request, only a police request is being raised instead of ambulance + police
6- When I raise a fire request, only a police request is being raised instead of police + Ambulance + fire

You will get tab to check this on tracking if you are creating these

**Comment by tariqarbi03** [comment]
The app is displaying the distance as 103.7 m for everyone using it and ETA 0.45 min for everyone using it. I am testing from Dubai and Madona is testing from Kuwait both for every request we raise we receive the above same ETA and Distance

this is because of AI model you can check in app as well you need to click long press on any button it will show logs
let me know if there is error

**Comment by chrisak86** [comment]
Issues 1 2 3 5 6 --> Fixed
Issue 4 The compass button on the map is not working**(fixed)**  --> Not yet fixed on both User and Police flow. 

Also when raising a request the map is showing kuwait city and not locating the user specifically. 
Please fix the location before after request is being raised to show user in the middle, mark the pointer location accurate and enable the compass button to work. The compass button should locate me even if I am away from the location and retrun the map to pointer location 


![WhatsApp Image 2024-09-20 at 12 44 23_433ca244](https://github.com/user-attachments/assets/d5607e33-7429-449e-bdf3-24caa0771408)

**Comment by tariqarbi03** [comment]
let me check again

**Comment by tariqarbi03** [comment]
we need to hide compass button because we are not using paid API for google map...

**Comment by chrisak86** [comment]
okk you can hide the button, but make sure to keep the location centered to where the accident happened not to show something else.

**Comment by tariqarbi03** [comment]
yes it is taking location of user .. i print lat long and checked if location permission is not enabled then he will show kuwait location but it will take user current location

**Comment by tariqarbi03** [comment]
if permission is accepeted it will show your location pointer as i showed you already else it will show kuwait but geolocator will take location of currently place

**Comment by chrisak86** [comment]
ok got it thanks

**Comment by tariqarbi03** [comment]
fixed ... Compass button removed

**Comment by chrisak86** [comment]
Compass button still their
